Sources/Noora/Components/YesOrNoChoicePrompt.swift

```diff
@@ -88,7 +88,7 @@ struct YesOrNoChoicePrompt {
 
         let yes = if answer {
             if terminal.isColored {
-                " Yes (y) ".onHex(theme.muted)
```

Comment:
Should we change or remove the `muted` color if it's unusable in light terminals – at least until we can support changing colors based on the terminal color?

Reply:
It's usable when you use it to represent fainted text. It should not be used as a background color like we were doing.